### ISSUE TITLE
Problems with sending messages

### ISSUE THREAD

**Issue by parthadc9** [AsamK/signal-cli#870]
java.lang.NullPointerException: Cannot invoke "java.util.Locale.getLanguage()" because "locale" is null
	at org.whispersystems.signalservice.internal.push.http.AcceptLanguagesUtil.getAcceptLanguageHeader(AcceptLanguagesUtil.java:19)

**Comment by AsamK** [comment]
A full strlack trace would be helpful.
This should already be fixed in version 0.10.2

**Comment by parthadc9** [comment]
The captcha authorisation was fine which followed with a Authorization failed error.... I tried to send a message and got this....

**Comment by MichaelBitard** [comment]
Probably related to https://github.com/AsamK/signal-cli/issues/833

I'm having the same issue right now, but since I'm using dbus, I don't have the full stacktrace, only this:

```
Error org.freedesktop.dbus.exceptions.DBusExecutionException: Error Executing Method org.asamk.Signal.sendGroupMessage: Cannot invoke "java.util.Locale.getLanguage()" because "locale" is null
```

**Comment by MichaelBitard** [comment]
I can confirm this is fixed in 0.10.2
